host/xtest/xtest_1000.c

```diff
@@ -440,9 +441,19 @@ static FILE *open_ta_file(const TEEC_UUID *uuid, const char *mode,
 			  bool for_write)
 {
 	char buf[PATH_MAX];
+	FILE *ret;
 
 	uuid_to_full_name(buf, sizeof(buf), uuid, for_write);
-	return fopen(buf, mode);
+	ret = fopen(buf, mode);
+	if (ret && for_write)
+		if (chmod(buf, 0666)) {
+			fprintf(stderr, "chmod(\"%s\"): %s",
+					buf, strerror(errno));
+			printf("chmod error! This test will fail if " \
+				"tee-supplicant does not have permission " \
+				"to load this TA file!\n");
```

Comment:
You're mentioning read permission by `tee-supplicant` but the subject of the commit is "Make xxx writable". I don't understand.